ibis/expr/signature.py

```diff
@@ -182,7 +182,7 @@ def __call__(cls, *args, **kwargs):
         return instance
 
 
-class Annotable(Hashable, util.EqMixin, metaclass=AnnotableMeta):
+class Annotable(Hashable, metaclass=AnnotableMeta):
```

Comment:
Was this supposed to inherit from `Comparable`?

Reply:
It was, but not anymore. Now we can use composition to make a class both [cached]comparable and annotable.